### PR TITLE
Feature/bc 229 exchange private task into draft

### PR DESCRIPTION
**Pull request by monaweitzenberg**
# Description
exchange text private task into draft

## Links to Tickets or other pull requests
https://ticketsystem.hpi-schul-cloud.org/browse/BC-229

## Changes
exchange text private task into draft

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: locales/es.json [hunk 1]
```diff
@@ -988,8 +988,7 @@
 			},
 			"tasks": {
 				"button": {
-					"createTask": "Crear una tarea",
-					"privateTasks": "Tareas privadas"
```
Comment (luizzaly) [inline locales/es.json:992]: Don't we need a translation?

File: locales/en.json [hunk 2]
```diff
@@ -988,8 +988,7 @@
 			},
 			"tasks": {
 				"button": {
-					"createTask": "Create a task",
-					"privateTasks": "Private tasks"
```
Comment (luizzaly) [inline locales/en.json:992]: why is it deleted?

File: locales/es.json (hunk 1 above)
Comment (luizzaly) [inline locales/es.json:992]: Why is the key deleted? (privateTasks)

Comment (monaweitzenberg) [reply locales/es.json:992]: the key is not in used. so I cleaned it up.

**Review by luizzaly — DISMISSED** [review]

File: locales/de.json [hunk 3]
```diff
@@ -3105,7 +3103,7 @@
 			},
 			"headline": {
 				"createTopic": "Thema anlegen",
-				"privateTasks": "Private Aufgaben",
+				"draftTasks": "Entwürfe",
 				"savedTeachingMaterials": "Gespeicherte Unterrichtsmaterialien"
 			},
 			"input": {
```
Comment (odalys-dataport) [inline locales/de.json:3109]: A few lines down, in line 3116/3114 there's an `"expandPrivateTasks": "Private Aufgaben ein\/ausklappen"` left

File: views/topic/topic.hbs [hunk 4]
```diff
@@ -144,12 +144,7 @@
                                 <small data-toggle="tooltip" title="{{$t "topic._topic.label.expandTasks"}}"><i
                                         class="fa fa-chevron-right collapse-icon" aria-hidden="true"></i></small>
                                 {{$t "global.headline.assignedTasks" }}
-                                <small>({{arrayLength homeworks}}
-                                    {{#ifneq 1 (arrayLength homeworks)}}
-                                        {{$t "global.headline.tasks"}}
-                                    {{else}}
-                                        {{$t "global.headline.task"}}
-                                    {{/ifneq}})
+                                <small>({{arrayLength homeworks}})
```
Comment (odalys-dataport) [inline views/topic/topic.hbs:147]: Why was the headline deleted here?

File: views/topic/topic.hbs [hunk 5]
```diff
@@ -167,21 +162,16 @@
...
                                         class="fa fa-chevron-right collapse-icon" aria-hidden="true"></i></small>
-                                {{$t "topic._topic.headline.privateTasks" }}
-                                <small>({{arrayLength myhomeworks}}
-                                    {{#ifneq 1 (arrayLength myhomeworks)}}
-                                        {{$t "global.headline.tasks"}}
-                                    {{else}}
-                                        {{$t "global.headline.task"}}
-                                    {{/ifneq}})
+                                {{$t "topic._topic.headline.draftTasks" }}
+                                <small>({{arrayLength myhomeworks}})
```
Comment (odalys-dataport) [inline views/topic/topic.hbs:174]: Same here, we're deleting the headling?

File: views/topic/topic.hbs [hunk 6]
```diff
@@ -167,21 +162,16 @@
                         {{#userHasPermission "HOMEWORK_CREATE"}}
                             <a href="/homework/new?course={{../courseId}}&topic={{../_id}}&private=true"
                                 class="btn btn-secondary btn-sm pull-right ml-1" data-toggle="tooltip"
-                                title="{{$t "global.button.addPrivateTask"}}">
-                                <i class="fa fa-plus" aria-hidden="true"></i><span class="hidden-md-down"> {{$t "global.button.addPrivateTask" }}</span>
+                                title="{{$t "global.button.addDraftTask"}}">
+                                <i class="fa fa-plus" aria-hidden="true"></i><span class="hidden-md-down"> {{$t "global.button.addDraftTask" }}</span>
```
Comment (odalys-dataport) [inline views/topic/topic.hbs:166]: In `topics.hbs` in lines 224 & 225 translation key `"global.button.addPrivateTask"` still needs to be changed

File: views/topic/topic.hbs [hunk 7]
```diff
@@ -167,21 +162,16 @@
...
                             </a>
                         {{/userHasPermission}}
                         <a data-toggle="collapse" class="collapse-toggle" href="#myhomeworks" tabindex="0">
                             <h2 class="h4">
                                 <small data-toggle="tooltip" title="{{$t "topic._topic.label.expandPrivateTasks" }}"><i
```
Comment (odalys-dataport) [inline views/topic/topic.hbs:171]: `expandPrivateTasks` needs to be changed

**Review by odalys-dataport — CHANGES_REQUESTED** [review]

File: views/topic/topic.hbs (hunk 4 above)
Comment (monaweitzenberg) [reply views/topic/topic.hbs:147]: Assigned tasks (1 task) vs  Assigned tasks (1)
From a UX point of view, it is sufficient to only specify the number. Please also consider a mobile view.

File: views/topic/topic.hbs (hunk 5 above)
Comment (monaweitzenberg) [reply views/topic/topic.hbs:174]: Private tasks (0 task) vs Private tasks (0)
From a UX point of view, it is sufficient to only specify the number. Please also consider a mobile view.

File: views/topic/topic.hbs (hunk 6 above)
Comment (monaweitzenberg) [reply views/topic/topic.hbs:166]: I changed it.

File: views/topic/topic.hbs (hunk 7 above)
Comment (monaweitzenberg) [reply views/topic/topic.hbs]: It's fixed.

**Review by odalys-dataport — APPROVED** [review]

**Pull request merged**